photon-lib/src/generate/photonlib.json.in

```diff
@@ -2,7 +2,8 @@
   "fileName": "photonlib.json",
   "name": "photonlib",
   "version": "${photon_version}",
-  "uuid": "515fe07e-bfc6-11fa-b3de-0242ac130004 ",
+  "uuid": "515fe07e-bfc6-11fa-b3de-0242ac130004",
+  "frcYear": "2024",
```

Comment:
we might also want to replace this instead of updating it every year?

Reply:
Maybe? I'll see how hard it is

Reply:
ez